src/modules/Builder/features/BuilderAppletSettings/BuilderAppletSettings.utils.tsx

```diff
@@ -33,16 +33,18 @@ export const getSettings = ({
             label: 'usersAndData',
             items: [
               {
+                name: 'exportData',
                 icon: <Svg id="export" />,
-                label: 'exportData',
+                label: t('exportData'),
```

Comment:
It seems like there are excess wraps with 't' func
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/4319881/7635775f-3dde-48d8-9fb3-a0bf6dfa535a)
In NavigationMenu/LeftBar it already exists